src/cli/serve/serve.js

```diff
@@ -185,7 +185,7 @@ export default function (program) {
       pluginPathCollector,
       []
     )
-    .option('--plugins <path>', 'an alias for --plugin-dir', pluginDirCollector)
+    .option('--plugins <path>', 'Deprecated, an alias for --plugin-dir', pluginDirCollector)
```

Comment:
Is this deprecation surfaced in the logs or UA UI?

Reply:
Not directly (no cli options are directly surfaced in UA), but this populates `plugins.scanDirs` that surfaces in UA.

Reply:
> Not directly (no cli options 'deprecations' are directly surfaced in UA or in logs), but this populates plugins.scanDirs

But it's not the same as deprecating `--plugins`. Is it? Maybe we should mention `--plugins` flag in `plugins.scanDirs` depraction message?